src/test/java/io/strimzi/kafka/metrics/DataPointSnapshotBuilderTest.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.metrics;
+
+import io.prometheus.metrics.model.snapshots.InfoSnapshot;
+import io.prometheus.metrics.model.snapshots.Labels;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DataPointSnapshotBuilderTest {
+
+    @Test
+    public void testNewLabelDuplicateWarning() {
+        Labels labels = Labels.builder().label("k1", "v1").label("k2", "v2").build();
+        InfoSnapshot.InfoDataPointSnapshot snapshot = DataPointSnapshotBuilder.infoDataPoint(labels, "value", "k1");
```

Comment:
I think it would be interesting to also test with labels that are not equal but collide when turned into Prometheus labels, for example `k-1` and `k_1`

Reply:
Just like in the other tests?
`assertEquals("k_1", PrometheusNaming.sanitizeLabelName("k-1"));`

Reply:
No, I mean using label names in `Labels.builder().label()` and `DataPointSnapshotBuilder.infoDataPoint()` that are not equal but collide. Currently we use `k1` in both.

Reply:
Changed to also test with labels that are not equal but collide when turned into Prometheus labels